build/_build/Build.Steps.cs

```diff
@@ -339,7 +339,7 @@ partial class Build
        .Executes(() =>
        {
            // start by copying everything from the tracer home dir
-           CopyDirectoryRecursively(TracerHomeDirectory, DDTracerHomeDirectory, DirectoryExistsPolicy.Merge);
+           CopyDirectoryRecursively(TracerHomeDirectory, DDTracerHomeDirectory, DirectoryExistsPolicy.Merge, FileExistsPolicy.OverwriteIfNewer);
```

Comment:
I think we should use `FileExistsPolicy.Overwrite` instead. Files _should_ always be newer, but in the unlikely case they wouldn't be, I'm afraid some files would be overwritten and other not, leading to inconsistencies that will be hard to figure out.

Reply:
Sounds good. Should I fix the other uses of `FileExistsPolicy.OverwriteIfNewer` as well?

Reply:
Changed them all in 7a8bee1519e525a6cd6eb33184480b1e55a4d2c3